span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGenerator.java

```diff
@@ -302,8 +305,7 @@ void populateOtherFields(
     // set scheme if specified
     fieldGeneratorMap.put(
         OTelHttpSemanticConventions.HTTP_SCHEME.getValue(),
-        (key, keyValue, builder, tagsMap) ->
-            builder.getRequestBuilder().setScheme(keyValue.getVStr()));
+        (key, keyValue, builder, tagsMap) -> setScheme(builder, tagsMap));
```

Comment:
we need to register same `setScheme` handler for `HTTP_REQUEST_FORWARDED` and `HTTP_REQUEST_X_FORWARDED_PROTO` too. See the USER_AGENT example.
- https://github.com/hypertrace/hypertrace-ingester/blob/main/span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGenerator.java#L90
- https://github.com/hypertrace/hypertrace-ingester/blob/main/span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGenerator.java#L308

Reply:
Secondly, we had decided to stop using first-class fields. So, we will need to built same logic here as well 
- https://github.com/hypertrace/hypertrace-ingester/blob/ece226386ff8b31ee9b1cf88444efbdb84e63750/semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java#L406

Reply:
Is this change for `SpanTypeAttributeEnricher`, right?

Reply:
@ankitchoudhary111 we no longer use the first-class field (http / grpc field generator). So, we need to move this logic to the right Semantic Utils method that is used the rest of the places.